extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -163,5 +160,27 @@ amp-carousel .amp-carousel-button.amp-disabled {
   order: 100000000 !important;
   margin-right: -1px !important;
 }
-
 /* End SlideScroll */
+
+
+
+
+
+/* Begin ScrollableCarousel */
+.-amp-scrollable-carousel-container .amp-carousel-slide {
+  display: inline-block !important;
```

Comment:
this is needed , else carousel would break is the page author makes it display: block/none/flex etc